modules/adbutlerBidAdapter.js

```diff
@@ -25,13 +25,15 @@ export const spec = {
     let requestURI;
     let serverRequests = [];
     let zoneCounters = {};
+    let extraParams = {};
 
     for (i = 0; i < validBidRequests.length; i++) {
       bidRequest = validBidRequests[i];
       zoneID = utils.getBidIdParameter('zoneID', bidRequest.params);
       accountID = utils.getBidIdParameter('accountID', bidRequest.params);
       keyword = utils.getBidIdParameter('keyword', bidRequest.params);
```

Comment:
You should be reading kw from fpd.context when available.

Reply:
Hi @patmmccann 

Thanks for the advice, that seems really handy! 

Perhaps that should be a separate commit & pull request since it's unrelated to this change? I'd be dropping that in as a replacement to the 'keyword' bid param on line 34 there.

Reply:
makes sense ty! I brought it up bc kw was an example of your extra param.